src/analysis/calculus/times_cont_diff.lean

```diff
@@ -981,9 +981,9 @@ lemma times_cont_diff_on_iff_continuous_on_differentiable_on {n : with_top ℕ}
   (hs : unique_diff_on 𝕜 s) :
   times_cont_diff_on 𝕜 n f s ↔
   (∀ (m : ℕ), (m : with_top ℕ) ≤ n →
-    continuous_on (λ x, iterated_fderiv_within 𝕜 m f s x) s)
+    continuous_on ((iterated_fderiv_within 𝕜 m f s)) s)
   ∧ (∀ (m : ℕ), (m : with_top ℕ) < n →
-    differentiable_on 𝕜 (λ x, iterated_fderiv_within 𝕜 m f s x) s) :=
+    differentiable_on 𝕜 ((iterated_fderiv_within 𝕜 m f s)) s) :=
```

Comment:
are the extra parens necessary? (I'm guessing they're autogenerated?)

Reply:
(the same comment applies to the next few lines of the same file. I didn't see potentially-redundant parentheses anywhere else.)